server/src/routes/user.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import userController from "../controllers/user";
+
+const router = Router();
+
+router.get("/:id", (req, resp) => userController.get(req, resp));
+router.put("/", (req, resp) => userController.update(req, resp));
+router.delete("/", (req, resp) => userController.remove(req, resp));
```

Comment:
I messed up again. Both of these per REST should have the `id` in the URL, regardless of the fact they send a body as well that might have that info.

Reply:
I know, I fixed that just now in subject branch so they all follow REST Api conventions. Will do that for the rest 😁👍